projects/packages/jetpack-mu-wpcom/src/features/wpcom-siteurl/siteurl.ts

```diff
@@ -0,0 +1,82 @@
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Adds the Site Address (URL) and WordPress Address (URL) input fields to the General Settings page on Simple sites.
+ */
+const wpcomAddSiteUrl = () => {
+	if ( typeof window.wpcomSiteUrl === 'undefined' ) {
+		return;
+	}
+
+	const settingsTable = document.querySelector( 'table.form-table tbody' );
+	if ( ! settingsTable ) {
+		return;
+	}
+
+	const previousSibling =
+		settingsTable.querySelector( '.site-icon-section' ) ||
+		settingsTable.querySelectorAll( 'tr' )[ 1 ];
+	if ( ! previousSibling ) {
+		return;
+	}
+
+	// Create the Site Address (URL) row
+	if ( ! document.getElementById( 'home' ) ) {
+		const homeUrlRow = document.createElement( 'tr' );
+		homeUrlRow.innerHTML = `
+					<th scope="row"><label for="home">Site Address (URL)</label></th>
+					<td><input type="url" id="home" value="${ window.wpcomSiteUrl.homeUrl }" class="regular-text code disabled" disabled="disabled" /></td>
```

Comment:
`home` as id here as it might conflict with other elements. I think it's good to use a specific id such as `homeUrl`

Reply:
Makes sense. Thanks for pointing!